pkg/stacks/templates/engines/helm2.go

```diff
@@ -0,0 +1,262 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package engines
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/go-logr/logr"
+	batchv1 "k8s.io/api/batch/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/yaml"
```

Comment:
I think you want `github.com/ghodss/yaml` here.  This will also remove `sigs.k8s.io/yaml` from go.sum.

Throughout crossplane we use this import for yaml marshalling.  Whenever I get lackluster (and incomplete)  unmarshalling, I end up discovering that I was using the sigs.k8s.io/yaml.

Reply:
Ooo, interesting. I'll have to look into this.

Reply:
I've made this change!